README.md

```diff
@@ -11,22 +11,711 @@
         <img src="https://img.shields.io/badge/code%20style-black-000000.svg" alt="Codestyle Black" height="17">
     </a>
 </p>
+
 # ClashOfClansAPI </br>
 Python Wrapper for SuperCells Clash Of Clans API </br>
 Get Token from https://developer.clashofclans.com/</br>
 
-# Install using pip </br>
+# Install
 
 > pip3 install cocapi
 
```

Comment:
@2kofawsome do you think we should add a `python setup.py install` or `pip install <PACKAGE-NAME>` ?

Reply:
I don't think I've ever used `python setup.py install`  so I would not want to add documentation about using it. I think having `> pip3 install cocapi` is enough, but you may want to add the instructions for `python setup.py install`

Reply:
Cool